### ISSUE TITLE
What batch size is assumed for the recommended learning rate from get_lr()?

### ISSUE THREAD

**Issue by GyoukChu** [thinking-machines-lab/tinker-cookbook#565]
[Supervised Learning Hyperparameters](https://tinker-docs.thinkingmachines.ai/supervised-learning/sl-hyperparams)

```
from tinker_cookbook.hyperparam_utils import get_lr
model_name = "meta-llama/Llama-3.1-8B-Instruct"
recommended_lr = get_lr(model_name, is_lora=False)
print(f"Recommended LR: {recommended_lr}")
```

**What is the corresponding batch size of this recommended LR? There is no information about that one.**

The document mentions $LR \propto \sqrt{B}$, but there is no information on the batch size at which the recommended lr is.

**Comment by YujiaBao** [comment]
The recommended LR from `get_lr()` was calibrated at **batch_size=128**.

We're working on updating and releasing the full results of the LR sweep — stay tuned!
